build.sbt

```diff
@@ -321,6 +321,18 @@ lazy val library = configureAsSubproject(project)
     products in Compile in packageBin ++=
       (products in Compile in packageBin in forkjoin).value,
     Osgi.headers += "Import-Package" -> "sun.misc;resolution:=optional, *",
+    packageOptions in (Compile, packageBin) +=  {
```

Comment:
This won't have any effect because it modifies the default packaging options but `scala-library.jar` gets built with BND, not with sbt's built-in packaging task. The right JAR can be built by `library/packageBin::packagedArtifact`, you can run `publishDists` to build a full distribution (with output similar to the ant build's). The correct place to add this should be `Osgi.bundleTask` (perhaps as a new setting or by honoring `packageOption` there, so it can be turned on specifically for `scala-library.jar` without affecting other projects).